api/src/Entity/Heist.php

```diff
@@ -218,14 +218,8 @@ class Heist
     private ?Location $location = null;
 
     #[Groups([self::CREATE])]
-    #[Assert\NotBlank(groups: [self::CREATE], message: 'heist.latitude.not_blank')]
-    #[Assert\Type(groups: [self::CREATE], type: 'float', message: 'heist.latitude.invalid')]
-    private ?float $latitude = null;
-
-    #[Groups([self::CREATE])]
-    #[Assert\NotBlank(groups: [self::CREATE], message: 'heist.longitude.not_blank')]
-    #[Assert\Type(groups: [self::CREATE], type: 'float', message: 'heist.longitude.invalid')]
-    private ?float $longitude = null;
+    #[Assert\NotBlank(groups: [self::CREATE], message: 'heist.place_id.not_blank')]
+    private ?string $placeId = null;
```

Comment:
Rajouter un commentaire pour expliquer rapidement pourquoi c'est là

```suggestion
    /**
      * Used to get or create the location of the heist on create, not mapped
      */
    #[Assert\NotBlank(groups: [self::CREATE], message: 'heist.place_id.not_blank')]
    private ?string $placeId = null;
```